.travis.yml

```diff
@@ -10,12 +13,23 @@ before_script:
     --with-jpeg-libraries=$JPEG/lib
     --with-magick=$WITH_MAGICK
   - make -j$JOBS -s 
+  - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
 script:
   - make -j$JOBS -s -k V=0 VERBOSE=1 check
   - LD_LIBRARY_PATH=$PWD/libvips/.libs
     DYLD_LIBRARY_PATH=$PWD/libvips/.libs
     LD_PRELOAD=$ASAN_DSO
     $PYTHON -m pytest -v test/test-suite
+  - docker build --build-arg VIPS_VERSION -t vips/vips-web .
+  - docker tag vips/vips-web:latest vips/vips-web:$VIPS_VERSION
+
+deploy:
+  provider: script
+  script: 
+    - docker push vips/vips-web:latest
+    - docker push vips/vips-web:$VIPS_VERSION
+  on:
+    branch: master
```

Comment:
Adding `tags: true` here will ensure we only deploy tagged commits and therefore stable software.

Reply:
Done